lib/diagnostic.rb

```diff
@@ -105,15 +122,24 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+    puts "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# A lot of this seems semantic to me: while you can get the same behavior
```

Comment:
The primary advantage is that classes can include any number of modules but can only inherit from one class.